### PR TITLE
[rock-dkms-bin] Reversed rock-dkms-bin/PKGBUILD version

### PR DESCRIPTION
**Pull request by vsauer**
==> Making package: rock-dkms-firmware-bin 3.5-2 (Mon 29 Jun 2020 05:42:50 PM PDT)
==> Retrieving sources...
  -> Downloading rock-dkms-firmware_3.5-32_all.deb...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found
==> ERROR: Failure while downloading http://repo.radeon.com/rocm/apt/debian/pool/main/r/rock-dkms/rock-dkms-firmware_3.5-32_all.deb
    Aborting...
Error downloading sources: rock-dkms-firmware-bin

### CODE REVIEW

File: rock-dkms-bin/PKGBUILD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Maintainer: acxz <akashpatel2008@yahoo.com>
 pkgname=rock-dkms-bin
 pkgver=3.5
-_pkgver=$pkgver-32
+_pkgver=$pkgver-30
 pkgrel=4
```
Comment (acxz) [inline rock-dkms-bin/PKGBUILD:5]: `pkgrel` should be bumped up by 1 whenever package is updated when the `pkgver` is still the same. See: https://wiki.archlinux.org/index.php/PKGBUILD#pkgrel

Comment (vsauer) [reply rock-dkms-bin/PKGBUILD]: Done. I pushed my forked master, but I am kinda new to the process. I am not sure if I need to do something else.

File: rock-dkms-firmware-bin/PKGBUILD [hunk 2]
```diff
@@ -1,8 +1,10 @@
 # Maintainer: acxz <akashpatel2008 at yahoo dot com>
+# Contributor: Vinicius Sauer <vsauer at gmail.com>
+
 pkgname=rock-dkms-firmware-bin
 pkgver=3.5
-_pkgver=$pkgver-32
-pkgrel=2
+_pkgver=$pkgver-30
+pkgrel=4
```
Comment (acxz) [inline rock-dkms-firmware-bin/PKGBUILD:7]: This should only be bumped up by 1

File: rock-dkms-bin/PKGBUILD (hunk 1 above)
Comment (acxz) [reply rock-dkms-bin/PKGBUILD]: np

**Comment by acxz** [comment]
Closing this issue since @vsauer has been unresponsive for more than a week. Feel free to open this up again as needed tho.

**Pull request closed without merging**